templates/nextjs/src/lib.ts

```diff
@@ -1,14 +1,21 @@
 import { Account, BN, TESTNET_NETWORK_URL } from 'fuels';
 
+// The two environments for the dapp are local and testnet.
 export const Environments = {
   LOCAL: 'local',
   TESTNET: 'testnet',
 } as const;
 type Environment = (typeof Environments)[keyof typeof Environments];
 
+/**
+ * The current environment is determined by the
+ * `NEXT_PUBLIC_DAPP_ENVIRONMENT` environment variable.
+ * If it's not set, the default is `local`.
```

Comment:
Maybe a link to the guide might be beneficial here and in the `.env.example`?
https://docs.fuel.network/docs/fuels-ts/creating-a-fuel-dapp/deploying-a-dapp-to-testnet/

Reply:
Not sure tbh, we have the link to the guide in the readme for the template/project

Reply:
Good point 👌🏼